frontend/views/containers/chatroom/chat-mentions/chat-mentions-utils.js

```diff
@@ -87,9 +88,14 @@ function createRecursiveDomObjects (element: any): DomObject {
       nodeObj.children.push(createRecursiveDomObjects(child))
     }
 
-    nodeObj.children = nodeObj.children.filter(
-      child => child.tagName || (child.text || '').trim().length
```

Comment:
@taoeffect Had more thorough investigation for the original issue and found out `trim()` here was the cause of the issue. As far as I remember, this was added to remove unnecessary `\n` created from `[DOMParser: parseFromString()](https://developer.mozilla.org/en-US/docs/Web/API/DOMParser/parseFromString) API. So updated the logic here.